File: src/main/java/ejercicioJava/sistemaReservaHotel/Reserva.java
package ejercicioJava.sistemaReservaHotel;

public class Reserva {
    public Cliente cliente;
    public Habitacion habitacion;

    public Reserva(Cliente cliente, Habitacion habitacion) {
        this.cliente = cliente;
        this.habitacion = habitacion;
    }

    @Override
    public String toString() {
        return "Cliente: " + cliente.getNombre() + ", Numero de habitación: " + habitacion.getNumeroHabitacion();
    }
}

File: src/main/kotlin/ejercicioKotlin/sistemaReservaHotel/Reserva.kt
package org.example.ejercicioKotlin.sistemaReservaHotel

data class Reserva(val cliente: Cliente?,  val habitacion: Habitacion?) {

    override fun toString(): String {
        return "Cliente: ${cliente?.nombre}, Numbero de habitación: ${habitacion?.numeroHabitaciion}"
    }

}